driver.md

```diff
@@ -117,10 +141,12 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
            <nonce> NONCE </nonce>
            ...
          </account>
-         <touchedAccounts> _ => SetItem(MINER) </touchedAccounts>
+         <touchedAccounts> _ => .Set </touchedAccounts>
```

Comment:
A reminder why omitting the MINER here during loadTx is ok? I don't remember the reason it was included in the first place.

Reply:
I couldn't find anywhere in the current codebase how MINER is used in `touchedAccounts`. Also, EIP-2929 specifies that 
> When a transaction execution begins, `accessed_addresses` is initialized to include the `tx.sender`, `tx.to` (or the address being created if it is a contract creation transaction) and the set of all precompiles.

Reply:
Hmm... Are we confusing the `accessed_addresses` set with the `touchedAccounts` set? From what I gather, `touchedAccounts` exists so empty accounts can be deleted at the end of a transaction. `accessed_addresses` exists to track gas costs for various account related opcodes. Two separate purposes, so it should probably be two separate sets? @ehildenb What do you think?